cmd/root.go

```diff
@@ -27,9 +27,9 @@ var (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "scorecard",
-	Short: "Security scorecards!",
-	Long:  `A scorecard program!`,
+	Use:   "./scorecard --repo=<repo_url> [--checks=check1,...]",
+	Short: "Open Source Scorecards",
```

Comment:
Unsure if we are calling this "Security scorecards or Open Source Scorecards", but we can change this later i guess.

Reply:
Yeah either works